shared/js/content-scope/protections.js

```diff
@@ -34,7 +35,7 @@ export async function loadProtections () {
     ]
 
     for (const protectionName of protectionNames) {
-        const protection = import(`./${protectionName}-protection.js`).then(({ init, load, update }) => {
+        const protection = import(`./${protectionName}Protection.js`).then(({ init, load, update }) => {
```

Comment:
Might be worth catching the error here.

Reply:
I would _much_ prefer to have dash-separated-lowercase filenames here. Why should these files be the only camelCased files in the whole repo?